docs/guide/widgets.md

```diff
@@ -190,6 +190,67 @@ If the supplied text is too long to fit within the widget, it will be cropped (a
 There are a number of styles that influence how titles are displayed (color and alignment).
 See the [style reference](../styles/index.md) for details.
 
+## Focus & keybindings
+
+Widgets can have a list of associated key [bindings](../guide/input.md#bindings),
+which let them call [actions](../guide/actions.md) in response to key presses.
+
+A widget is only able to handle key presses if it or one of its descendants has [focus](../guide/input.md#input-focus).
```

Comment:
```suggestion
A widget is able to handle key presses if it or one of its descendants has [focus](../guide/input.md#input-focus).
```

omitting needless words. And it probably isn't the *only* way to handle keypresses...

Reply:
Removing the "only" changes the nuance of the sentence, so I wouldn't call it needless. You're right that it's not technically correct though - will fix!